Doc/library/sqlite3.rst

```diff
@@ -829,35 +829,55 @@ Exceptions
 
 .. exception:: Warning
 
-   A subclass of :exc:`Exception`.
+   Exception raised for important warnings like data truncations while
+   inserting, etc.  It is a subclass of :exc:`Exception`.
 
 .. exception:: Error
 
-   The base class of the other exceptions in this module.  It is a subclass
-   of :exc:`Exception`.
+   The base class of the other error exceptions in this module.  Use this to
+   catch all errors with one single :keyword:`except` statement.  :exc:`Error`
+   is a subclass of :exc:`Exception`.
+
+.. exception:: InterfaceError
+
+   Exception raised for errors that are related to the database interface
+   rather than the database itself.  It is a subclass of :exc:`Error`.
 
 .. exception:: DatabaseError
 
-   Exception raised for errors that are related to the database.
+   Exception raised for errors that are related to the database.  It is a
+   subclass of :exc:`Error`.
+
+.. exception:: DataError
+
+   Exception raised for errors that are due to problems with the processed data
+   like division by zero, numeric value out of range, etc.  It is a subclass of
```

Comment:
I don't know example when it is raised for division by zero. It can be raised for numeric value out of range and too long strings/blobs.

Reply:
I'll adjust the text. Thanks!